share/picongpu/examples/WarmCopper/cmakeFlags

```diff
@@ -29,8 +29,8 @@
 #   - start with zero index
 #   - increase by 1, no gaps
 
-flags[0]="-DALPAKA_CUDA_ARCH=20 -DCMAKE_DISABLE_FIND_PACKAGE_ISAAC=True"
-flags[1]="-DALPAKA_CUDA_ARCH=20 -DCMAKE_DISABLE_FIND_PACKAGE_ISAAC=True -DPARAM_OVERWRITES:LIST='-DPARAM_ENABLE_PUSHER=1;-DPARAM_ENABLE_CURRENT=1'"
+flags[0]="-DCMAKE_DISABLE_FIND_PACKAGE_ISAAC=True"
```

Comment:
Only a question which is not related to this PR: Why do you disable ISAAC explicit?

Reply:
I can't remember, probably to cut compile time for this example :)